R/check_code.R

```diff
@@ -30,16 +34,22 @@
 #' @template learnr_args
 #' @template glue_correct
 #' @template glue_incorrect
-#' @param glue_pipe A glue string that returns the final message displayed when a user uses a pipe,
+#' @param glue_pipe A glue string that returns the final message displayed when
+#'    the studentuses a pipe,
 #'    \code{$>$}. Defaults to \code{getOption("gradethis_glue_pipe")}.
 #'
 #' @return a \code{\link{graded}} object.
 #'   An incorrect message will describe the first way that the answer differs,
 #'   the message will be the content of the \code{glue_pipe} argument.
 #'
+#' @seealso \code{\link{check_code}}, \code{\link{check_result}}, and \code{\link{test_result}}
 #' @export
 #' @examples
 #' \dontrun{grading_demo()}
+#'
+#' y <- quote(sqrt(log(2)))
+#' z <- quote(sqrt(log(1)))
+#' check_code(grader_args = list(user_quo = y, solution_quo = z))
```

Comment:
add a comment about this being a manual example.  "See grading demo for tutorial usage."

Reply:
but this would be how someone would use `gradethis` as a standalone library

Reply:
Added comment about learnr tutorial